app/views/questions/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<% provide :title, t(".title") %>
+<h1><%= t ".header" %></h1>
+
+<div class="row">
+  <div class="col-md-8 col-md-offset-2">
+    <div id="new-question">
+      <%= form_for(@question) do |f| %>
+        <%= render "shared/error_messages", object: f.object %>
+        
+        <div class="form-group">
+          <%= f.label :subject, "Subject" %>
+          <%= collection_select :subject, :name, Subject.all, :name, :name,
+            {}, {class: "form-control"} %>
+        </div>
+
+        <div class="form-group">
+          <%= f.label :question, "Question" %>
+          <%= f.text_area :content, class: "form-control" %><br>
+        </div>
+        
+        <div class="form-group">
+          <%= f.label :question_level, "Difficulty" %><br>
+
+          <%= f.radio_button :level, "easy", checked: true %>
+          <%= f.label :level, "Easy", value: "easy" %>
+        
+          <%= f.radio_button :level, "normal" %>
+          <%= f.label :level, "Normal", value: "normal" %>
+
+          <%= f.radio_button :level, "hard" %>
+          <%= f.label :level, "Hard", value: "hard" %>
+        </div>
+
+        <div class="form-group">
+          <%= f.label :question_type, "Question type" %><br>
+        
+          <%= f.radio_button :question_type, "single_choice", checked: true %>
```

Comment:
`"single_choice"` -> lấy key của enum type trong question chú ơi

Reply:
đấy chính là key ạ

Reply:
key của enum type thì chú lấy giá trị từ key của enum chứ ko settings string thế này

https://stackoverflow.com/questions/38668069/rails-5-get-enum-key-via-integer-value
https://stackoverflow.com/questions/45374174/how-to-get-key-name-from-a-enum-value-in-rails/45374287